backend/src/main/java/heartbeat/handler/AsyncExceptionHandler.java

```diff
@@ -0,0 +1,37 @@
+package heartbeat.handler;
+
+import heartbeat.exception.BaseException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+
+@Component
+@RequiredArgsConstructor
+public class AsyncExceptionHandler {
+
+	private final Map<String, BaseException> exceptionMap = new ConcurrentHashMap<>();
+
+	private static final Long EXPORT_CSV_VALIDITY_TIME = 1800000L;
```

Comment:
Just export into a single file and store it `EXPORT_CSV_VALIDITY_TIME`, and shared with the other files